lib/samples/manage_features/manage_features.dart

```diff
@@ -0,0 +1,433 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+class ManageFeatures extends StatefulWidget {
+  const ManageFeatures({super.key});
+
+  @override
+  State<ManageFeatures> createState() => _ManageFeaturesState();
+}
+
+class _ManageFeaturesState extends State<ManageFeatures> {
+  // Create a controller for the map view.
+  final _mapViewController = ArcGISMapView.createController();
+
+  // Create a service feature table.
+  late final ServiceFeatureTable _damageServiceFeatureTable;
+  // Create a feature layer.
+  late final FeatureLayer _damageFeatureLayer;
+  // The name of the attribute being used in the sample.
+  final damageTypeFieldName = 'typdamage';
+
+  // Create a list of feature management options.
+  final _featureManagementOptions =
+      <DropdownMenuItem<FeatureManagementOperation>>[];
+  // Create a variable to store the selected operation.
+  FeatureManagementOperation? _selectedOperation;
+
+  // Create a list of damage type attribute options.
+  final _damageTypeAttributeOptions = <DropdownMenuItem<String>>[];
+  // Create a variable to store the attribute value of the selected damage type.
+  String? _selectedDamageType;
+
+  // Create a variable to store the selected feature.
+  Feature? _selectedFeature;
+
+  // A flag for when the map view is ready and controls can be used.
+  var _ready = false;
+
+  @override
+  void initState() {
+    super.initState();
+    // Add each feature management operation to the list of dropdown menu options.
+    FeatureManagementOperation.values
+        .map(
+          (operation) => _featureManagementOptions.add(
+            DropdownMenuItem(
+              onTap: () => setState(
+                () => _selectedOperation == operation,
+              ),
+              value: operation,
+              child: Text(getLabel(operation)),
+            ),
+          ),
+        )
+        .toList();
```

Comment:
I'm not sure if it matters, but I would flip this around a bit like so:
```dart
_featureManagementOptions.addAll(
  FeatureManagementOperation.values.map(
    (operation) => DropDownMenuItem(
      // ...
    ),
  ),
);
```
That way it reads a bit smoother to me and it saves you the trouble of having to call `.toList()`. It's a bit obscure that you have to call `.toList()` in order to make the calls to `_featureManagementOptions.add()` happen.